packages/core/core-flows/src/order/workflows/get-order-detail.ts

```diff
@@ -0,0 +1,159 @@
+import { OrderDTO, OrderDetailDTO } from "@medusajs/types"
+import { deduplicate } from "@medusajs/utils"
+import {
+  WorkflowData,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { useRemoteQueryStep } from "../../common"
+
+enum PaymentStatus {
```

Comment:
I am not sure what is best to do around enums, but at least for the http typings we said we'll just go with a string union instead. Now, for workflows and BE enums we can continue adding them to `utils`, or maybe we also go with string unions? WDYT?

At the very least, these should be in either `types` or `utils`, not inside the workflow

Reply:
The typings are using string union, here I just need these locally. Will bring it to the functions on the utils

Reply:
👍